### ISSUE TITLE
Interaction issues on mobile

### ISSUE THREAD

**Issue by wonderxboy** [wekan/wekan#452]
I see the following issues when using the board on iPhone and iPad 
1. Some places need tap twice to activate a "click", for example, the add label button on card detail view, and the a card need to be clicked twice to open the detail view 
2. Drag and drop not working

**Comment by floatinghotpot** [comment]
It's true, I also notice this issue.

**Comment by araczkowski** [comment]
hi,

since: https://github.com/wekan/wekan/issues/449 it should be OK

please try

```javascript
meteor update mquandalle:jquery-ui-drag-drop-sort
```

**Comment by floatinghotpot** [comment]
Drag & drop works now, but on card details view, still need touch twice to fire the click event.

**Comment by araczkowski** [comment]
I checked on my android device (nexus 5) it's ok - the click if fired after one touch 
the same in the browser, when the "Emulate mobile" is ON

maybe you have iPhone, or the browser cache has old js ... or I don't know ;) 


![image](https://cloud.githubusercontent.com/assets/1263968/11979446/c0c295e0-a993-11e5-85a2-704e3b50e449.png)

**Comment by floatinghotpot** [comment]
Same here, I have to touch the page first (change the focus?), then touch the link will work. It won't work if show the card details, then click the menu icon directly.

**Comment by nata-g** [comment]
Also, the sign-in and sign out pages are  not displaying properly on my Android Nexus 6P phone

**Comment by floatinghotpot** [comment]
@nata-g Can you paste a screenshot?

**Comment by nata-g** [comment]
@floatinghotpot  here you go....

These are the screenshots taken from my nexus 6p. I guess it might be the issue with how the language selection is displayed on the screen.

screenshot 1: before entering the user name.

![image](https://cloud.githubusercontent.com/assets/203255/12004910/c82a83a8-ab35-11e5-986b-44cef45c3e95.png)

screenshot2: when I start entering my user name


![image](https://cloud.githubusercontent.com/assets/203255/12004916/1727909a-ab36-11e5-8ac2-ed83720cb26b.png)

screenshot 3: when entering the password

![image](https://cloud.githubusercontent.com/assets/203255/12004918/4972f80a-ab36-11e5-8b79-b216c328e8c1.png)

screenshot4: after logging out

![image](https://cloud.githubusercontent.com/assets/203255/12004921/7aeb2c9a-ab36-11e5-99d8-c3a79ba39445.png)

Hopefully these help.
BTW, thanks for all that awesome contributions.  I am learning a lot from your  sick badassery contributions :smile: .

**Comment by floatinghotpot** [comment]
Okay, I got it, will fix it soon.

**Comment by floatinghotpot** [comment]
@nata-g I realise my PR #455 will fix the layout/css issue you reported, but won't fix other tap issue in this tracker. In future, if you report a different issue, it's better to create a new issue tracker.

**Comment by mquandalle** [comment]
@floatinghotpot is right, it’s better to keep one issue per thread. Let’s keep this one about “tap twice to activate a click” and use #448 for the issue with the login pages.

**Comment by wonderxboy** [comment]
@araczkowski @floatinghotpot I can confirm that drag & drop works now after #449 
However, the double tap issue still persists on many buttons including card details page and side bar.
Like @floatinghotpot mentioned, these buttons need on tap to be activated then the second tap to perform their real functionality.

PS: The issue does not appear on Chrome emulator with "Emulate Mobile" option on. However, it appears on the real deal - xcode iphone emulator or the real devices (iPhone 5s, 6, iPad air)

**Comment by nata-g** [comment]
Yes I can confirm the same issue too. For me it mainly happens on the cards, you have to tap twice to get in to the card detail page.

**Comment by wonderxboy** [comment]
Anyone has a good clue on how to fix this? It seems as if something has a hover state, which prevents the click to be activated at first try.

**Comment by neooleg** [comment]
> Some places need tap twice to activate a "click", for example, the add label button on card detail view, and the a card need to be clicked twice to open the detail view

Looks like this issue/thread can be moved to #953 (as well as #882 moved before by @xet7).

**Comment by xet7** [comment]
Moved to #953
